solutions/usaco-615.mdx

```diff
@@ -2,16 +2,53 @@
 id: usaco-615
 source: USACO Bronze 2016 February
 title: Milk Pails
-author: Danh Ta Chi Thanh
+author: Ryan Chou (Python), Danh Ta Chi Thanh (C++)
 ---
 
 [Official Analysis (Java)](http://www.usaco.org/current/data/sol_pails_bronze_feb16.html)
 
+## Implementation
+
 <LanguageSection>
+	
+<PySection>
+
+```python
+
+# Code by Ryan Chou
+
```

Comment:
```suggestion
```
your name is already in the authors list

Reply:
sorry about that, I forgot to delete it. Thanks for the suggestions!

Reply:
I've removed it.